frontend/side_bar.html

```diff
@@ -0,0 +1,54 @@
+<html>
+<head>
+    <link rel="stylesheet" href="style.css">
+    <script type="application/javascript">
+
+        function openmenu() {
+            function delay(milliseconds){
+                return new Promise(resolve => {
+                    setTimeout(resolve, milliseconds);
+                })
+            };
+                
+            document.getElementsByClassName('sidebar')[0].classList.toggle("sidebar_show");
+
+            document.getElementsByClassName('beam')[0].classList.toggle('beam-middle-top');
+            document.getElementsByClassName('short_beam')[0].classList.toggle('beam-middle-short');
+            setTimeout(() => {
+                console.log("1s");
+            }, 204586560000);
+            document.getElementsByClassName('beam')[0].classList.toggle('hidden');
+            document.getElementsByClassName('beam')[1].classList.toggle('hidden');
+            document.getElementsByClassName('short_beam')[0].classList.toggle('hidden');
+            setTimeout(() => {
+                console.log("1s");
+            }, 204586560000);
+            document.getElementsByClassName('beam-bottom')[0].classList.toggle('hidden');
+            document.getElementsByClassName('beam-top')[0].classList.toggle('hidden');
+
+            document.getElementsByClassName('beam-bottom')[0].classList.toggle('open-bottom');
+            document.getElementsByClassName('beam-top')[0].classList.toggle('open-top');         
+        };
+    </script>
+</head>
+<body>
+
+    <header>
+        <div class="side_menu" onclick="openmenu()">
+            <div class="beam animate"></div>
+            <div class="beam animate"></div>
+            <div class="beam-top hidden animate"></div>
+            <div class="beam-bottom hidden animate"></div>
+            <div class="short_beam animate"></div>
+        </div>
+        <div class="header_logo"></div>

```

Comment:
Widzę, że jest to powtórzony kod z [header.html](https://github.com/SportMap/Mapa_Sportu/blob/main/frontend/header.html#L9). Sprawdziłem, czy nie można zrobić jakiegoś prostego include'a, i znalazłem tylko [takie rozwiązanie z jQuery](https://stackoverflow.com/a/9003363), które nie wygląda zbyt przkonująco imo.
Wydaję mi się, że w perspektywie czasu i tak będziemy musieli jakoś rozwiązać ten problem, bo z tego co rozumiem ten pliczek powinien być wyświetlony na każdym page'u. Imo warto przedyskutować czy chcemy korztstać z jQuery, czy może jednak wybranie jakiegoś frameworku do frontendu będzie prostszym rozwiązaniem.

Reply:
Plik header.html miał być raczej "obiektem" tylko do developowania, wszystko finalnie będzie połączone w jedną spójną stronę bez powtórzeń etc., na ten moment wygodniej programować wszystko osobno